apt_info.py

```diff
@@ -147,13 +176,18 @@ def _write_reboot_required(registry):
 
 
 def _main():
+    logging.basicConfig(stream=sys.stderr)
```

Comment:
i think stream=sys.stderr is pretty much the default, why do we need to do this here?

Reply:
blah it is indeed. I'll remove the param